content/zh-cn/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -203,7 +199,7 @@ Any Pods exposed by a Service have the following DNS resolution available:
 
 一般而言，Pod 会对应如下 DNS 名字解析：
 
-`pod-ip-address.my-namespace.pod.cluster-domain.example`
+`pod-ip-address.my-namespace.pod.cluster-domain.example`.
```

Comment:
I think the ending period indicate the end of paragraph. In Mandarin maybe we can just ignore it.

(Also for Line 207)